config.yaml

```diff
@@ -14,6 +14,7 @@ permissions:
 commands:
   alert:
     executable: /home/bundle/cog-command
+    description: "Triggers alerts"
```

Comment:
Not a big deal, but might be nice to stick to the same verb style for all of these `Triggers alerts` vs `Trigger alerts` and `Acknowledge incidents` vs `Acknowledges incidents`.